ballerina-tests/constraint-validation-tests/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+org = "ballerina"
+name = "constraint_validation_tests"
+version = "0.1.0"
+
+[[dependency]]
```

Comment:
what is this dependency and why we need this?

Reply:
There are lot of integration tests in here. We use this test common package to store the common utility functions